odmlui/treemodel/ValueModel.py

```diff
@@ -109,6 +109,10 @@ def get_display(self, max_length=-1):
         typically takes the first line (max *max_length* chars) and adds '…'
         """
         text = str(self.pseudo_values)
+
+        # Always escape "&" and "<" since they break the view otherwise.
+        text = text.replace("&", "&amp;").replace("<", "&lt;")
```

Comment:
Are there any other characters we should look out for here?
Do you think it's worth using an HTML or the XML library to escape characters as necessary?